packages/react/src/QueeryKeeper.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import { FronteggProvider } from './FronteggProvider';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import * as QueryKeeper from './queryKeeper';
+import 'whatwg-fetch';
+
+describe('Query keeper', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('Should call useQueryKeeper when no external history is provided ', async () => {
+    const mockUseQueryKeeper = jest.spyOn(QueryKeeper, 'useQueryKeeper');
+    render(<FronteggProvider contextOptions={{ baseUrl: 'mock-base-url' }} />, { wrapper: BrowserRouter as any });
```

Comment:
this test is more relevant to FronteggProvider folder, because you check that FronteggProvider calls some QueryKeeper logic.